src/views/RecIM/views/Match/index.js

```diff
@@ -0,0 +1,102 @@
+import { Grid, Typography, Card, CardHeader, CardContent } from '@material-ui/core/';
+import { useParams } from 'react-router';
+import { useUser } from 'hooks';
+import GordonLoader from 'components/Loader';
+import GordonUnauthorized from 'components/GordonUnauthorized';
+import { ParticipantListing } from 'views/RecIM/components/Listing';
+import styles from './Match.module.css';
+import { first } from 'lodash';
+
+// CARD - main
+const MainCard = () => {
+  return (
+    <Card>
+      <CardContent>
+        <Grid container justifyContent="space-between">
+          <Grid item className={styles.grayText}>
+            League Name
+          </Grid>
+          <Grid item className={styles.grayText}>
+            Date/Time
+          </Grid>
+        </Grid>
+        <Grid container alignItems="center" justifyContent="space-around">
+          <Grid item xs={1}>
+            <Typography variant="h5">Team A</Typography>
+            <i className={styles.grayText}>Sportsmanship</i>
+          </Grid>
+          <Grid item xs={1}>
+            <img src={''} alt="Team Icon" width="85em"></img>
+          </Grid>
+          <Grid item xs={1}>
+            <Typography variant="body" className={styles.grayText}>
+              <i>Match status</i>
+            </Typography>
+            <Typography variant="h5">17 - 38</Typography>
+          </Grid>
```

Comment:
```suggestion
          <Grid item>
            <Grid container direction="column" justifyContent="center" alignItems="center">
              <Typography variant="body" className={styles.grayText}>
                <i>Match status</i>
              </Typography>
              <Typography variant="h5">17 - 38</Typography>
            </Grid>
          </Grid>
```

Reply:
<img width="1402" alt="Screen Shot 2022-10-14 at 1 37 04 PM" src="https://user-images.githubusercontent.com/78386128/195908676-3ef06367-5ceb-40d4-a07d-d81654ba9096.png">
Small alignment issue that was bugging me